app-text/searchmonkey/searchmonkey-2.0.0-r1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils qmake-utils
+
+MY_PN=${PN}2
+MY_P=${PN}_v${PV}
+
+DESCRIPTION="Powerful text searches using regular expressions"
+HOMEPAGE="http://searchmonkey.sourceforge.net/"
+# Switch to ^s when we switch to EAPI=6.
+SRC_URI="mirror://sourceforge/project/${PN}/S${MY_PN:1}/${PV}%20%5Bstable%5D/${MY_P}.zip"
+
+LICENSE="GPL-3"
+SLOT="2"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-qt/qtcore:4
+	dev-qt/qtgui:4"
+DEPEND="${RDEPEND}
+	app-arch/unzip"
+
+S="${WORKDIR}"/${MY_P}
+
+PATCHES=(
+	"${FILESDIR}"/${P}-gcc4.7.patch
+	"${FILESDIR}"/${P}-gcc6.patch
```

Comment:
Just one small thing (your EAPI 6 bumps in general are flawless though). For more "fluid" ebuilds (with actual upstream activity) I prefer including the versions, i.e. `"${FILESDIR}"/${PN}-2.0.0-gcc4.7.patch`, because it can take some releases before patches are integrated.